CMakeLists.txt

```diff
@@ -53,6 +53,12 @@ install(DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/ DESTINATION ${CMAKE_INSTALL
 install(DIRECTORY pd-help/ DESTINATION ${CMAKE_INSTALL_PREFIX})
 
 #Dependencies: availability
+#Non-standard openssl path on OSX
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
```

Comment:
That is a tricky thing.
We cannot hardcode the path to openssl - decides to put it somewhere else it breaks everything.

I think, we should go for let the user set OPENSSL_ROOT (if he needs it) and add a comment into the README.md.

`cmake -DOPENSSL_ROOT_DIR=/usr/local/ssl -COPENSSL_LIBRARIES=/usr/local/ssl/lib`

http://stackoverflow.com/questions/16248775/cmake-not-able-to-find-openssl

What do you think?